externals/kyuubi-spark-sql-engine/pom.xml

```diff
@@ -219,6 +221,20 @@
                                 <include>org.apache.curator.**</include>
                             </includes>
                         </relocation>
+                        <relocation>
+                            <pattern>org.apache.hive</pattern>
+                            <shadedPattern>${kyuubi.shade.packageName}.org.apache.hive</shadedPattern>
+                            <includes>
+                                <include>org.apache.hive.**</include>
+                            </includes>
+                        </relocation>
+                        <relocation>
+                            <pattern>org.apache.thrift</pattern>
+                            <shadedPattern>${kyuubi.shade.packageName}.org.apache.thrift</shadedPattern>
+                            <includes>
+                                <include>org.apache.thrift.**</include>
```

Comment:
fb303 has another package name

Reply:
what is that?

Reply:
Something related with thrift https://mvnrepository.com/artifact/org.apache.hive/hive-service-rpc/3.1.2